contracts/src/v0.8/functions/dev/v1_X/ocr/OCR2Base.sol

```diff
@@ -19,9 +19,6 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
     i_uniqueReports = uniqueReports;
```

Comment:
We were planning to remove unique reports, right?

Reply:
Right